openoa/analysis/yaw_misalignment.py

```diff
@@ -0,0 +1,571 @@
+        avergaged over all wind speed bins are listed. If UQ is True, 95% confidence intervals will
+        be plotted for the binned power performance values and listed for the yaw misalignment
+        estiamtes.
+
+        Args:
+            turbine_ids (list[str], optional): Name of turbines for which yaw misalignment data are
+                plotted. If None, plots for all turbines for which yaw misalginment detection was
+                performed will be generated. Defaults to None.
+            xlim (:obj:`tuple[float, float]`, optional): A tuple of floats representing the x-axis
+                wind vane angle plotting display limits (degrees). Defaults to (None, None).
+            ylim (:obj:`tuple[float, float]`, optional): A tuple of the y-axis plotting display limits
+                for the power performance vs. wind vane plots. Defaults to (None, None).
+            return_fig (:obj:`bool`, optional): Flag to return the figure and axes objects. Defaults to False.
+            figure_kwargs (:obj:`dict`, optional): Additional figure instantiation keyword arguments
+                that are passed to `plt.figure()`. Defaults to None.
+            plot_kwargs_curve (:obj:`dict`, optional): Additional plotting keyword arguments that are passed to
+                `ax.plot()` for plotting lines for the power performance vs. wind vane plots. Defaults to {}.
+            plot_kwargs_line (:obj:`dict`, optional): Additional plotting keyword arguments that are passed to
+                `ax.plot()` for plotting vertical lines indicating mean vane angle and vane angle where
+                power is maximized. Defaults to {}.
+            plot_kwargs_fill (:obj:`dict`, optional): If `UQ` is True, additional plotting keyword arguments
+                that are passed to `ax.fill_between()` for plotting shading regions for 95% confidence
+                intervals for power performance vs. wind vane. Defaults to {}.
+            legend_kwargs (:obj:`dict`, optional): Additional legend keyword arguments that are passed to
+                `ax.legend()` for the power performance vs. wind vane plots. Defaults to {}.
+        Returns:
+            None | dict of tuple[matplotlib.pyplot.Figure, matplotlib.pyplot.Axes]:
+                If `return_fig` is True, then a dictionary containing the figure and axes object(s)
+                corresponding to the yaw misalignment plots for each turbine are returned for further
+                tinkering/saving. The turbine names in the `turbine_ids` aregument are the dicitonary
+                keys.
+        """
+
+        if self._use_power_coeff:
+            power_performance_label = "Normalized Cp (-)"
+        else:
+            power_performance_label = "Normalized Power (-)"
+
+        if turbine_ids is None:
+            turbine_ids = self.turbine_ids
+        else:
+            if (set(turbine_ids) - set(self.turbine_ids)) != set():
```

Comment:
Pretty minor, but this can be done with Python's set math as follows:

```python
if (set(turbine_ids).difference(self.turbine_ids)):
```

Reply:
Updated